app/helpers/avo/turbo_stream_actions_helper.rb

```diff
@@ -11,7 +11,9 @@ def flash_alerts
     end
 
     def close_action_modal
-      turbo_stream_action_tag :replace, target: "actions_show", html: @view_context.turbo_frame_tag("actions_show")
+      turbo_stream_action_tag :replace,
+        target: Avo::ACTIONS_TURBO_FRAME_ID,
+        html: @view_context.turbo_frame_tag(Avo::ACTIONS_TURBO_FRAME_ID)
```

Comment:
Instead of doing a replace and going through the `view_context` and all, wouldn't it be easier to empty it?
`turbo_power` has `inner_html = ""`

Reply:
https://github.com/avo-hq/avo/pull/2521#issuecomment-1966017434